geniza/corpus/views.py

```diff
@@ -95,6 +95,9 @@ def get_queryset(self):
             DocumentSolrQuerySet()
             .filter(status=Document.PUBLIC_LABEL)
             .facet_field("type", exclude="type", sort="value")
+            .facet_field("has_digital_edition")
+            .facet_field("has_translation")
+            .facet_field("has_discussion")
```

Comment:
these facets can be added in a single call with `facet` if we don't need extra options like the exclusion and sort we have on type
```suggestion
           .facet("has_digital_edition", "has_translation", "has_discussion")
```